pkgs/applications/networking/cluster/talosctl/default.nix

```
@@ -1,11 +1,11 @@
 { lib, buildGoModule, fetchFromGitHub, installShellFiles }:
 let
   # look for GO_LDFLAGS getting set in the Makefile
-  version = "0.14.2";
-  sha256 = "sha256-sQtry94T5cDO+836D/p/8ptQi3WYKDBLr1QZyEXdLQI=";
-  vendorSha256 = "sha256-cd2iNMxWmkSWqqkPLYocUG+fCUXoeUXEuGQxjUWQnXk=";
-  pkgsVersion = "0.9.0-4-gc875fbe";
-  extrasVersion = "0.7.0-2-gb4c9d21";
+  version = "0.14.3";
+  sha256 = "sha256-toEbWUZxnJkUe9jkZRdJrcPXu+CIh62fUKyX38OkXxU=";
+  vendorSha256 = "sha256-Pj1918TIOGfhkRiFEKkURL4xMFgWroiNsNU6yWzT8yk=";
+  pkgsVersion = "v0.9.0-6-gbfcc795";
+  extrasVersion = "v0.7.0-2-gb4c9d21";
```

```suggestion
  pkgsVersion = "0.9.0-6-gbfcc795";
  extrasVersion = "0.7.0-2-gb4c9d21";
```
Otherwise it becomes `vv`.

https://github.com/NixOS/nixpkgs/blob/d5058ae9e5c1e45378f9eea6401c3c7368d3e688/pkgs/applications/networking/cluster/talosctl/default.nix#L32-L33

---

true - I dropped the `v` in L32-33, and kept it in the line above.

---

Would have preferred to follow the style that is used in nixpkgs by dropping the `v` here and adding it below.

Seems inconsistent to then do the opposite by having `rev = "v${version}";` and then use `${src.rev}"` on the next lines above.

---

This was mostly due to the comment referring to the makefile, and the `v` being very present there.

---

I'd be up to changing this back to having the `v` in fronts, once we have an updater script, but as of now, this seems to be the much less error-prone flavour.

---

Going against the predominate nixpkgs style seems likely to be error prone for the next people to edit/review to edit this ...

---

Unfortunately, I'm right now the only one in maintainers. I welcome a possible future maintainer to change this, or even better, provide an update script ;-)

---

I would add myself as maintainer but you just rejected my review feedback?

---

I didn't reject it per se, just didn't want to do it in this PR.

If you want to add yourself as a maintainer and provide an update script to help with this, we can remove the `v`'s from the front of this, and I'd gladly review such a PR.

---

> I welcome a possible future maintainer to change this, or even better, provide an update script

> If you want to add yourself as a maintainer and provide an update script to help with this, we can remove the v's from the front of this

The update script wasn't a requirement before and now it is?

---

> I welcome a possible future maintainer to change this

> just didn't want to do it in this PR

This is very confusing.